cedar-lean/Cedar/Thm/Data/LT.lean

```diff
@@ -55,7 +55,7 @@ theorem IPNet.lt_conn {a₁ a₂ p₁ p₂ : Nat}
 
 instance IPNet.strictLT : StrictLT Ext.IPAddr.IPNet where
   asymmetric a b   := by
-    cases a <;> cases b <;> simp [LT.lt, Ext.IPAddr.IPNet.lt]
+    cases a <;> cases b <;> simp only [LT.lt] <;> simp only [Ext.IPAddr.IPNet.lt, UInt32.val_val_eq_toNat, Nat.lt_eq, Bool.or_eq_true, decide_eq_true_eq, Bool.and_eq_true, not_false_eq_true, not_true_eq_false, imp_self]
```

Comment:
I wonder if we should be defining some custom tactics for long `simp only` clauses. It seems like the instances in this file at least have a lot of overlap.

Reply:
Seems like a good idea but I don't know how to do it quickly and want to get this merged now so I'll open an issue: https://github.com/cedar-policy/cedar-spec/issues/279